projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -0,0 +1,91 @@
+@import 'mixins';
+
+:host {
```

Comment:
Could we do this without using host? A div inside? This prevents a user from setting a width for their own use case. Discussed here https://github.com/hypertrace/hypertrace-core-ui/pull/57#discussion_r464719695

Reply:
I copied the base component from the select component. working on cleaning up things now. We don't need the host here. Its left over from using material components inside select I think.